src/offline_hand_eye/README.md

```diff
@@ -148,14 +148,14 @@ to print the extrinsic calibration in the console, e.g.
 
 ```
 At time 0.0
-- Translation: [-0.659, -0.459, 1.551]
-- Rotation: in Quaternion [0.695, 0.604, -0.255, 0.295]
-- Rotation: in RPY (radian) [2.997, 0.789, 1.371]
-- Rotation: in RPY (degree) [171.701, 45.233, 78.552]
+- Translation: [0.658, 0.461, 1.540]
+- Rotation: in Quaternion [-0.610, 0.698, 0.279, 0.250]
+- Rotation: in RPY (radian) [3.026, 0.760, -1.751]
```

Comment:
@td-code these values look a bit different (than what we are using in the `festo_cobot_model.urdf.xacro` and I believe this is the result of my (inaccurate) quaternion rotation (in `handeye_calibration.json`). I would stick to 
```
<origin xyz="0.659 0.459 1.550" rpy="2.999 0.787 -1.771"/>
```
and I can make another attempt to rotate the quaternion with more accuracy...